### DISCUSSION TITLE
Support for selects and input sizes

### DISCUSSION THREAD

**Discussion by dbrdarski** [swordev/suid#162] [General]
Hi, I really liked your project so far, but I TextField component does not support different sizes (or dense option)
and there is no select component (or support for selects via TextField). Do you have an idea if and when these features might be added? I understand that this library is WIP and I can work my way around current limitations, but found those two missing features to be what this project misses to be in some form "complete" at least at this round of iteration.

**Comment by juanrgm** [comment]
Hi,

Thanks for reporting the `size` bug (https://github.com/swordev/suid/commit/9d5860d16549e21ac8b326ab61091ed4ff3c3f6e).

The `Select` component is a priority for me (that's why I finished the Menu components recently), but also there are other priorities as bug fixes, SSR support, SolidJS updates, MUI updates and other issues and overall.

I value the feedback and thanks again.
